src/handlers/store-standardised-invoices/store-line-item.ts

```diff
@@ -32,7 +34,14 @@ export async function storeLineItem(
 
   await putTextS3(
     bucket,
-    `${destinationFolder}/${getStandardisedInvoiceFileName(bodyObject)}`,
+    getStandardisedInvoiceKey(destinationFolder, bodyObject),
+    record.body
+  );
+
+  // TODO The legacy storage location.  This will go away with BTM-486.
+  await putTextS3(
+    bucket,
+    `${legacyDestinationFolder}/${getStandardisedInvoiceFileName(bodyObject)}`,
```

Comment:
Correct, but I think this will be a rare occurrence and using the full set of subfolders won't be necessary.  The date will be part of the filename so there should be no confusion.